components/Layout.js

```diff
@@ -28,6 +28,8 @@ export default function Layout({ children }) {
           isExtendedNavList={isExtendedNavList}
           session={session}
           currentPath={currentPath}
+          role="navigation"
```

Comment:
when you Navlist is a `<nav>` element, you don't need to specify this role, since this is already set as the default value for this element.